pydal/_gae.py

```diff
@@ -83,3 +78,6 @@ def _validate(self, value):
                 return decimal.Decimal(value)
             raise TypeError("Property %s must be a Decimal or string."
                             % self._name)
```

Comment:
And why is that a problem? If the first line of the `try` block fails the interpreter will stop the execution and jump to the `except` one, no matter of how many lines you have inside the block..